pkg/operator/v1helpers/interfaces.go

```diff
@@ -19,7 +19,7 @@ type OperatorClient interface {
 }
 
 type StaticPodOperatorClient interface {
-	OperatorClient
+	OperatorClientWithFinalizers
```

Comment:
Why this change, I'm not seeing it being used here nor in https://github.com/openshift/cluster-kube-controller-manager-operator/pull/543

Reply:
It is being used in https://github.com/openshift/library-go/pull/1095/commits/1df083fadac8328066bb845a4d6049a2870742c5#diff-d0ce926df996bfa572967a0d2a1d011bbfc84cd41e065b957328285647b17cecR293, technically deploymentController started using it and I had to modify the methods to be implemented in staticPodOperatorClient :( but I think in the long run this good as we can provide OperatorClientWithFinalizers for all the staticpodClients.